backend/lib/index.js

```diff
@@ -0,0 +1,145 @@
+const { Repo } = require('hologit/lib')
+const handlebars = require('handlebars')
+const csvParser = require('csv-parser')
+const TOML = require('@iarna/toml')
+
+module.exports = class GitSheets {
+  static async create(gitDir) {
```

Comment:
should this be `gitDir=null` ?

Reply:
It's not necessary (since it would just be `undefined` if not passed) but it would make it more self-documenting, yes — I'll update it.

Reply:
I run a typescript linter in my vscode while I work on js and it complains about these sorts of things so I've gotten in the habit of spotting it